src/types.py

```diff
@@ -0,0 +1,3 @@
+from pathlib import Path
+
+Command = list[str | Path]
```

Comment:
The `| Path` part seems to lead to casts to `str` in at least `write_test_script`. Would it be simpler to have this be a list of `str`, delete this file, then have casts to `Path` when necessary?

Reply:
Going to keep as is. This is used for type hinting only and makes my linter not hate me.

Some of the casts from `Path -> str` can be done with `my_path.name` and I will look to see if any code can be made cleaner this way.